dstep/translator/Type.d

```diff
@@ -201,8 +201,13 @@ package string reduceAlias(Type type)
         case "u32": return "uint";
         case "u64": return "ulong";
 
-        default: return null;
+        default: break;
     }
+
+    if (type.spelling == "byte" && type.canonical.kind == CXTypeKind.uChar)
```

Comment:
I think it would be better to have this go through the `translateType`:

https://github.com/jacob-carlborg/dstep/blob/0bba701beb5413037d93a03a45b4b58207a04e91/dstep/translator/Type.d#L552

Reply:
Fixed, unfortunately it required passing context is some more places.